api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -634,4 +636,19 @@ private boolean userHasEverLoggedIn(
     userService.confirmPublications();
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
+
+  /**
+   * Get a JSON list of users and their registered tier access expiration dates.
+   *
+   * <p>This endpoint is intended as a temporary manual measure to assist with user communication
+   * during the rollout of Annual Access Renewal (AAR).
+   *
+   * <p>Once fully rolled out, we will have an automated expiration email process and this can
+   * likely be removed. See RW-6689 and RW-6703.
+   */
+  @Override
+  @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
```

Comment:
myself learning purpose: What is ACCESS_CONTROL_ADMIN? I have seens few *Admins in Authority but not sure how we tell the difference. Feels we have too many admins types

Reply:
This is the authority which can manipulate user data and can do things like bypass access modules.

We split up the different admin abilities as an attempt to meet the Principle of Least Privilege.

see workbench-api.yaml for more:
```
      Authorities are a tool used to add discrete permissions to users. We currently
      define the following authorities:
        * REVIEW_RESEARCH_PURPOSE: This authority allows the user to review workspaces where
          the user has requested review to approve or reject.
        * DEVELOPER: Grants access to some internal-only
          administrative endpoints, including group management and runtime
          management. Intended to be granted for devops needs, e.g. for oncall.
        * ACCESS_CONTROL_ADMIN: This is actually basically a user admin authority, for people
          to perform actions on a user's enabled status and manual verification.
        * FEATURED_WORKSPACE_ADMIN: Allows a user to publish workspaces
        * COMMUNICATIONS_ADMIN: Allows a user to post a Status Alert banner
        * SECURITY_ADMIN: Allows a user to respond to egress alerts by deleting all runtimes in a project
        * INSTITUTION_ADMIN: Allows a user to create, delete, and update Institute entities
        * RESEARCHER_DATA_VIEW: Allows a user to view admin-only data (or aggregations of data)
            for users, workspaces, and similar user-created content.
```